properties/P000140.md

```diff
@@ -7,7 +7,12 @@ aliases:
 refs:
   - mr: MR2048350
     name: General Topology (Willard)
+  - mathse: 4646084
+    name: Unraveling the various definitions of k-space or compactly generated space
 ---
 
-Subsets are closed (resp. open) whenever their intersection with every compact
+Subsets are closed (resp. open) exactly when their intersection with every compact
 subset is closed (resp. open) in the subspace topology.
+
+Equivalently, subsets are closed (resp. open) exactly when their intersection with every continuous
+image of a compact space is closed (resp. open) in the subspace topology.
```

Comment:
Shouldn't this be instead that a subset $U\subseteq X$ is open in $X$ if for every continuous map $f:K\to X$ with $K$ an arbitrary compact space, the inverse image $f^{-1}(U)$ is compact in $K$?  And same thing for P141.  Maybe we could mention "final topology" also?

Reply:
"image $f^{-1}(U)$ is *open* in K" I think, yes. I was being lazy when copy-pasting definitions and didn't think about this carefully.

I think these are equivalent when $f$ is an inclusion map from a subspace; it's less clear that these are equivalent when $f$ is not injective (and I suspect they aren't).